eldritch/events.py

```diff
@@ -1806,6 +1816,10 @@ def resolve(self, state):
       return
 
     encounters = [card.encounter_event(self.character, world_name) for card in self.cards]
+    if any(isinstance(enc, Unimplemented) for enc in encounters):
+      self.draw = None
```

Comment:
Does this work? Because when you get here, len(self.cards) == self.draw_count, so you won't draw another card the next time resolve() is called. And you haven't removed anything from cards, so the player can still select an unimplemented encounter. I think you would need to put the cards back in the deck (maybe only put the unimplemented cards back?) before doing this.

Reply:
Given that "ignore cards that aren't implemented" isn't a proper feature of original rules, I think removing all cards (not just the un implemented ones) is fine, and saves having to muck about with the middles of lists.